question-service/README.md

```diff
@@ -0,0 +1,51 @@
+# PeerPrep Backend for Question Service - Developer Readme 
+
+## Quick Start
+
+- This backend service uses <b>npm</b> as the package manager
+- Why not yarn? Because I was too lazy to switch to yarn
+
+### How to run backend server
+
+1. Add the `QUESTION_SERVICE_MONGODB_URL` (which zac sent in tele chat) into the `.env` file
+2. Run `npm run dev` (make sure you are in the `backend` directory)
```

Comment:
something minor: before this step new devs need to run `npm i` to install all the dependencies

Reply:
oki donezo